### ISSUE TITLE
Add duplicate filtering

### ISSUE THREAD

**Issue by Randl** [mtalcott/google-photos-deduper#7]
It would be convenient to be able to filter duplicates by parameters, e.g., show only those with different resolutions, those with the same name, or those with the same size, etc.

**Comment by mtalcott** [comment]
Agreed! I'd also like to add some sorting by number of duplicates, max/min similarity.
